util/Migrator/DbScripts/2023-11-28_00_Send_Add_CipherId_Column.sql

```diff
@@ -0,0 +1,145 @@
+-- Add CipherId column
+IF COL_LENGTH('[dbo].[Send]', 'CipherId') IS NULL
+BEGIN
+    ALTER TABLE [dbo].[Send] 
+        ADD [CipherId] UNIQUEIDENTIFIER NULL,
+        CONSTRAINT [FK_Send_Cipher] FOREIGN KEY ([CipherId]) REFERENCES [dbo].[Cipher]([[Id]]);
+END
+GO
+
+-- Recreate View
+IF OBJECT_ID('[dbo].[SendView]') IS NOT NULL
+BEGIN
+    DROP VIEW [dbo].[SendView]
+END
+GO
+
+CREATE VIEW [dbo].[SendView]
+AS
+SELECT
+    *
+FROM
+    [dbo].[Send]
+GO
+
+CREATE OR ALTER PROCEDURE [dbo].[Send_Create]
+    @Id UNIQUEIDENTIFIER,
+    @UserId UNIQUEIDENTIFIER,
+    @OrganizationId UNIQUEIDENTIFIER,
+    @Type TINYINT,
+    @Data VARCHAR(MAX),
+    @Key VARCHAR(MAX),
+    @Password NVARCHAR(300),
+    @MaxAccessCount INT,
+    @AccessCount INT,
+    @CreationDate DATETIME2(7),
+    @RevisionDate DATETIME2(7),
+    @ExpirationDate DATETIME2(7),
+    @DeletionDate DATETIME2(7),
+    @Disabled BIT,
+    @HideEmail BIT,
+    @CipherId UNIQUEIDENTIFIER
```

Comment:
❌ The null setting needs to be here in the script too, as this is what applies the migration.